package.json

```diff
@@ -5,9 +5,9 @@
   "main": "dist/sofe.js",
   "scripts": {
     "test": "karma start",
-    "build": "babel src --out-dir lib && node build.js",
+    "build": "npm run clean && babel src --out-dir lib && node build.js",
```

Comment:
`clean` before `build` because deleted files in the `src` directory can still exist in the `lib` directory unless you clean first